msctl

```diff
@@ -1731,20 +1734,22 @@ queryStart() {
   SERVER_PID=$(getJavaPID $1)
   # Delete any old query.in or query.out buffer files.
   rm -Rf "$WORLD_DIR/query.in" "$WORLD_DIR/query.out"
+  # Initialize the query.in and query.out buffer files.
```

Comment:
This bit should not be here.  Merge failure on my part, I'll submit a new patch..